Sources/_StringProcessing/Regex/AnyRegexOutput.swift

```diff
@@ -20,21 +23,22 @@ public struct AnyRegexOutput {
 
 @available(SwiftStdlib 5.7, *)
 extension AnyRegexOutput {
-  /// Creates a type-erased regex output from an existing match.
+  /// Creates a dynamic regular expression match output from an existing match.
```

Comment:
Are we settling on "dynamic" for this? I thought we were leaning more towards "type-erased" at one point

Reply:
Oh, right — great question. I made this change because while "type-erased" is the way we look at `AnyRegexOutput` as implementors, for `Regex` users the experience of using a `Match<AnyRegexOutput>` is more like it's a dynamic type that they can query. Typically type erasure results in difficulty accessing the underlying values, which isn't as much the case here due to the different `AnyRegexOutput` API.